src/payment/create-payment-strategy-registry.ts

```diff
@@ -253,5 +254,15 @@ export default function createPaymentStrategyRegistry(
         )
     );
 
+    registry.register('masterpass', () =>
```

Comment:
Does this work with all Masterpass providers? i.e.: Braintree and Stripe? Do you require any provider-specific logic?

Reply:
We expect to use the same logic for all providers at least on the front-end